curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64ec9282cd547785258cecf2.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-17
 
 # --description--
 
-Using arrow syntax, create an implicit return and use addition assignment to set the `innerHTML` of `tasksContainer` to empty backticks.
+Using arrow syntax, use addition assignment to set the `innerHTML` of `tasksContainer` to empty backticks.
```

Comment:
```suggestion
Using arrow syntax, complete the `forEach` callback function. Inside the callback function body, use an addition assignment operator to set the `innerHTML` of `tasksContainer` to empty backticks.
```

Reply:
@jdwilkin4 changes the description for step 17

Reply:
It looks like you didn't include the backticks `` when you implemented this change.
That is what the latest review comment is about 👍

Reply:
Hi @jdwilkin4 , i missed that things , included in my next commit